grid.v3/client_test.go

```diff
@@ -63,6 +66,14 @@ func (a *echoActor) Act(c context.Context) {
 	}
 }
 
+type EchoMsg struct {
+	Msg string
+}
+
```

Comment:
`GobCodecRegister` is a convenience method that generates a GOB codec for a type and registers it.